packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -219,6 +256,66 @@ - (UIImage *)scaleImage:(UIImage *)image by:(id)scaleParam {
   return image;
 }
 
+/**
+ * Creates a scaled version of the provided UIImage based on a specified scale factor. If the scale
+ * factor is significantly different from the image's current scale, a new UIImage object is
```

Comment:
What does "significantly" mean in this context?
Maybe rephrase a little saying if the scale factor is almost identical (a tiny epsilon-delta is ignored) to the current scale, return the original image.